gateleen-playground/src/main/resources/playground/server/admin/v1/routing/rules

```diff
@@ -1,4 +1,8 @@
 {
+  "/playground/service/unavailable/(.*)": {
```

Comment:
You don't need a capture group. I.e. it's enough to write `/playground/service/unavailable/.*`

Reply:
Correct. Since all other rules in this resource have a capture group, I leave this as it is to keep consistency